### ISSUE TITLE
Consider releasing a 1.0.0

### ISSUE THREAD

**Issue by PowerKiKi** [mroth/lolcommits#375]
This project is now [more than 6 years old](https://github.com/mroth/lolcommits/blob/master/CHANGELOG.md#001-29-march-2012) and should probably be considered "production ready" at this point. As the [project claims to follow semver](https://github.com/mroth/lolcommits/blob/master/CHANGELOG.md#lolcommits-change-log), you should probably consider releasing the next version as 1.0.0 to convey the maturity of the project and follow [semver rules](https://semver.org/#how-do-i-know-when-to-release-100).

**Comment by matthutchinson** [comment]
Good point! @mroth how do you feel about bumping the version to `1.0` - now that I've extracted all the plugins. Or was that something you wanted to hold off doing until this (now old) [refactoring PR](https://github.com/mroth/lolcommits/pull/343) is complete?

**Comment by mroth** [comment]
Hmm.  I do agree we're close to the time we should do a `1.0`.  Looking at #343 I do also think that if we can make it happen, doing so before a 1.0 release is the right time, because afterwards we'll want to keep the format fairly consistent. #343 would also greatly increase the usability I think, so maybe let's review it, see what it would take, and either go forward with it and reject it.

Off the top of my head, the road to v1.0 stuff would then be:
- [ ] Review and merge/reject #343
- [ ] Documentation/README/website/etc cleanup
- [ ] Finally figure out how to move core lolcommits project itself to the lolcommits organization, to reflect that this really is an effort outside of me now and that others get proper credit! :-)
- [ ] Once we're at 1.0-ish, I'd like to consider a homebrew package to make it easy for folks to install (I'm pretty sure I know how to do this now that we're good on deps). 

@matthutchinson What do you think?  (We could also make a GH "project" or "milestone" or whatever it is to track towards this.)

**Comment by matthutchinson** [comment]
Project created:  https://github.com/mroth/lolcommits/projects/2

**Comment by matthutchinson** [comment]
Also - I'll see whats required to resolve the conflicts on #343
